tools/scripts/generate_relay/main.go

```diff
@@ -0,0 +1,341 @@
+// generate_relay.go - Tool for generating RelayRequest data and running load tests
+//
+// This tool generates a properly formatted RelayRequest, creates a Lua script for wrk2,
+// and executes load testing against a RelayMiner endpoint using kubectl.
+//
+// Usage:
+//   go run tools/scripts/generate_relay/main.go [flags]
+//
+// Flags:
```

Comment:
I like where your head is at.

Do you think we could embed this in `pocketd relayminer relay`?

Reply:
This script depends on `k8s` and `wrk2`, while `pocketd relayminer relay` is more standalone.

Also, `wrk2` nad `pocketd relayminer relay` seem to have overlapping purposes.

I don't see how we can have such integration :thinking: